### PR TITLE
new-question

### PR DESCRIPTION
**Pull request by ayushk12**
changes the text of the question.

### CODE REVIEW

File: javascript/answers/q6.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+    </div>
+
+    <div>
+        <p>
+            <small>
+
+                <a href="../../index.html">Home</a>
+                &gt; <a href="../index.html">JavaScript Questions</a>
+                &gt; What are escape Sequences?
```
Comment (arpansac) [inline javascript/answers/q6.html]: Change the text of the question

File: javascript/answers/q6.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        gtag('config', 'UA-157837769-1');
+    </script>
+
+    <title>
+        JavaScript | InterQues | Coding Ninjas
+    </title>
+
+    <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png" />
+
+    <meta name="description" content="What are escape Sequences?">
```
Comment (arpansac) [inline javascript/answers/q6.html]: Please change the question text

**Review by arpansac — CHANGES_REQUESTED** [review]

File: javascript/answers/q6.html (hunk 1 above)
Comment (ayushk12) [reply javascript/answers/q6.html]: changes the text of the question done.

File: javascript/index.html [hunk 3]
```diff
@@ -111,6 +111,10 @@ <h4>
                 <a href="./answers/Dhanu084-q5.html">
                     <h4>What are escape Sequences?</h4>
                 </a>
+            </li>
+            <li>
+                <a href="./answers/ayushk12-q1.html"><h4>What is the value of !'bang'?</h4></a>
```
Comment (shivamyth) [inline javascript/index.html]: It's not q1, take a fresh pull and update the q(no.)

Comment (ayushk12) [reply javascript/index.html]: updated

**Review by arpansac — APPROVED** [review]
Great! Something new to learn

**Pull request merged**